text-ui-test/runtest.bat

```diff
@@ -15,6 +15,6 @@ for /f "tokens=*" %%a in (
 java -jar %jarloc% < ..\..\text-ui-test\input.txt > ..\..\text-ui-test\ACTUAL.TXT
 
 cd ..\..\text-ui-test
-rmdir trippie_data /Q /S >NUL
+if exist trippie_data del trippie_data
```

Comment:
`del` is used to delete file. Based on `runtest.sh` looks like you wanted to delete a directory and its content, right? Use `rmdir /q /s trippie_data` instead, it is equivalent to `rm -rf`

Reply:
Since your PC OS is macOS and mine is Windows (Error is in .bat file which is executable in Windows). I will take-over this error.

Reply:
Thank you @felixhalim

Reply:
I tried to delete a directory, and output no errors if it does not exist.